public/app/features/alerting/unified/components/expressions/Expression.tsx

```diff
@@ -158,20 +162,25 @@ export const Expression: FC<ExpressionProps> = ({
         </div>
         {hasResults && (
           <>
-            <ExpressionResult series={series} isAlertCondition={isAlertCondition} />
-
-            <div className={styles.footer}>
-              <Stack direction="row" alignItems="center">
-                <Spacer />
-
-                <PreviewSummary
-                  isCondition={Boolean(isAlertCondition)}
-                  firing={groupedByState[PromAlertingRuleState.Firing].length}
-                  normal={groupedByState[PromAlertingRuleState.Inactive].length}
-                  seriesCount={seriesCount}
-                />
-              </Stack>
-            </div>
+            <ExpressionResult
+              series={series}
+              isAlertCondition={isAlertCondition}
+              isRecordingRule={isGrafanaRecordingRule}
```

Comment:
Wouldn't be easier to just pass `type` into the components?

Reply:
what would be the benefit?

Reply:
Mostly clarity. This prop only gives us information about recording rules but nothing about other supported types
I have seen a few places with code checking if this is recording rule and I thought that maybe it would be easier to just pass type instead

Reply:
ok! makes sense if in the future we add different logic for other types 👍